src/test/java/io/antmedia/integration/AppFunctionalV2Test.java

```diff
@@ -237,6 +238,59 @@ public void testSetUpEndPoints() {
 		}
 
 	}
+	
+	@Test
+	public void testSetUpEndPointsV2() {
+
+		try {
+			RestServiceV2Test restService = new RestServiceV2Test();
+
+			Broadcast source=restService.createBroadcast("source_stream");
+			Broadcast endpointStream=restService.createBroadcast("endpoint_stream");
+
+			Endpoint endpoint = new Endpoint();
+			endpoint.setRtmpUrl(endpointStream.getRtmpURL());
+			
+			restService.addEndpointV2(source.getStreamId(), endpoint);
+
+			Thread.sleep(1000);
+
+			assertNotNull(restService.getBroadcast(source.getStreamId()).getEndPointList());
+
+			Process rtmpSendingProcess = execute(ffmpegPath
+					+ " -re -i src/test/resources/test.flv  -codec copy -f flv rtmp://127.0.0.1/LiveApp/"
+					+ source.getStreamId());
+
+			//wait for fetching stream
+			Thread.sleep(5000);
```

Comment:
Please don't use thread.sleep if you don't have to.

Reply:
We need this because It takes some time to create the mp4 file.